recipes/hdfs3/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: hdfs3
```

Comment:
If possible, I'd prefer to leave this package named `hdfs3` here to match our github repo, docs, pypi, etc. and keep things simple, where `hdfs3` is the Python library that wraps `libhdfs3`. There are packages on pypi named `hdfs`, `pywebhdfs`, `py4hdfs`, `python-hdfs`, `webhdfspy`, `pywebhdfs`, and so on, I think it would be confusing to have `hdfs3` on pypi and `python-hdfs3` on conda-forge.